docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Do you want to **report a bug?** or **request a feature?** 
+
+## Report a bug
+
+Please provide the steps to reproduce and if possible a reproductible environment. **As we are working directly with container, please provide the Dockerfile sample or the Docker image name**
```

Comment:
Please provide the steps to reproduce your problem and, if possible, a full reproducible environment. **As we are working directly with containers, please provide the Dockerfile sample or the Docker image name**